tests/data/config/err1.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this?

Reply:
Artifact of adding a line break to files that didn't end with one. Guess for empty file that's fine. Reverted it here